src/music/element/MusicQueue.py

```diff
@@ -9,8 +9,11 @@
 class MusicQueue:
     """Reperesents a Queue GUI object. Handles which MusicSource to play next and displays in the GUI.
     """
-    def __init__(self, active_embed: ChatEmbed, client: discord.Client, list: List[MusicSource] = []) -> None:
-        self.list = list
+    def __init__(self, active_embed: ChatEmbed, client: discord.Client, playlist: List[MusicSource] = None) -> None:
+        if playlist == None:
```

Comment:
Not a biggie, but to make the code cleaner you can probably do this.
self.playlist =[]
if playlist:
  self.playlist = playlist